app/controllers/ExpensesController.scala

```diff
@@ -58,15 +43,12 @@ class ExpensesController @Inject()(cc: ControllerComponents,
           (json \ "amount").asOpt[Double].map { amount =>
             (json \ "note").asOpt[String].map { note =>
               (json \ "categoryId").asOpt[String].map { categoryId =>
-                (json \ "сreatedDate").asOpt[String].map { dateString =>
+                (json \ "createdDate").asOpt[Long].map { dateTimestamp =>
                   try {
-                    val obj = new Expenses(id, userId, amount, note, categoryId, new SimpleDateFormat("dd/M/yyyy hh:mm").parse(dateString))
+                    val obj = new Expenses(id, userId, amount, note, categoryId, new Date(dateTimestamp))
```

Comment:
@Bukarachka It's pretty strange to use entity classes in controllers, should be extracted, Please create a ticket for that and refactor in future.